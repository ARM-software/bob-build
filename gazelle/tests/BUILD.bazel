load("@bazel_gazelle//:def.bzl", "gazelle_generation_test")

[gazelle_generation_test(
    name = file[0:-len("/WORKSPACE")],
    size = "small",
    gazelle_binary = "//:gazelle_bin",
    test_data = glob(
        include = [file[0:-len("/WORKSPACE")] + "/**"],
    ),
) for file in glob(["**/WORKSPACE"])]
