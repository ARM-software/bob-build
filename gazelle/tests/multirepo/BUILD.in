# gazelle:exclude ignored
# gazelle:exclude repo1/ignored
