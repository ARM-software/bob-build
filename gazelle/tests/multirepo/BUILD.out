# gazelle:bob_ignore ignored
# gazelle:bob_ignore repo1/ignored
